src/Entity/ApplicationRequest/ApplicationRequest.php

```diff
@@ -413,4 +420,14 @@ public function getFavoriteThemesAsString(): string
 
         return implode(', ', $themes);
     }
+
+    public function getTakenForCity(): ?string
+    {
+        return $this->takenForCity;
+    }
+
+    public function setTakenForCity(?string $takenForCity = null): void
```

Comment:
a setter should never be called with `no` argument.

so `(?string $takenForCity)` would be enough. (no need a default value if no parameter is given)